src/js/oTable.js

```diff
@@ -0,0 +1,182 @@
+/* global exports, document, HTMLElement, CustomEvent, setImmediate */
+
+// TODO: Use Dom-Deletgate -- https://www.npmjs.com/package/dom-delegate
```

Comment:
So we can add events to yet to be added components? I'm not really sure, we seem to add it to other components.